hearpreprocess/pipeline.py

```diff
@@ -607,12 +714,16 @@ def requires(self):
         }
 
     def run(self):
-        for audiofile in tqdm(list(self.requires()["corpus"].workdir.glob("*.wav"))):
-            # Compare the filename with the slug.
-            # Note that the slug does not have the extension of the file
-            split = self.metadata.loc[
-                self.metadata["slug"] == audiofile.stem, "split"
-            ].values[0]
+        audiofiles = set(self.requires()["corpus"].workdir.glob("*.wav"))
+        for audiofile in audiofiles:
```

Comment:
We lost the tqdm bar here -- can we add that back in?

Reply:
It's just symlinks tho, this step is really fast. So why clutter the luigi output?